lib/config.js

```diff
@@ -40,7 +49,128 @@ module.exports.getConfig = async function getConfig({
     config.replacers = []
   }
 
-  config['sort-direction'] = validateSortDirection(config['sort-direction'])
+  if (error) throw error
 
   return config
 }
+
+const category = Joi.object()
+  .keys({
+    title: Joi.string().required(),
+    label: Joi.array()
+      .items(Joi.string())
+      .single(),
+    labels: Joi.array()
```

Comment:
Not sure we want to keep both

Reply:
perhaps worth making a breaking change when we also have published the the JSON schema?

Reply:
We keep can both in the schema.

Reply:
should they both allow single? perhaps labels should not?